platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/datasource/VirtualDataSource.java

```diff
@@ -87,6 +87,33 @@ void saveRecords(
             final Stream<VirtualLeafRecord<K, V>> leafRecordsToDelete)
             throws IOException;
 
```

Comment:
Does it make sense to change the existing `saveRecords()` method in `VirtualDataSource` to be a default method (calling the new method with "false" as the last arg)?

Reply:
Yes, it's a good idea, thank you